src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,38 @@
+package duke.parser;
+
+/**
+ * Class for parsing user input.
+ */
+public class Parser {
+    public Parser() {
+    }
+
+    /**
+     * Gets a command from the user.
+     *
+     * @param input original String input from user
+     * @return an integer that represents a command
+     */
+    public int getCommand(String input) {
+        String[] i = input.split("\\s");
+        if (i[0].equals("bye")) {
```

Comment:
Have you considered using a switch statement? In a sense, they are equivalent to each other. But switch is used for long if-else statement where the conditions are similar.